redi-connect-front/src/components/organisms/Checklist.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Section, Container, Element, Columns } from 'react-bulma-components'
+import { useTranslation } from 'react-i18next'
+import Heading from '../atoms/Heading'
+import './Checklist.scss'
+
+interface Props {
+  type: 'mentor' | 'mentee'
+}
+
+const Checklist = ({ type }: Props) => {
+  const { t } = useTranslation()
+
+  return (
+    <Section className="default-background">
+      <Container>
+        <Element
+          renderAs="h4"
+          textAlignment="centered"
+          textTransform="uppercase"
+          textSize={6}
+          responsive={{ mobile: { textSize: { value: 7 } } }}
+        >
+          {t(`loggedOutArea.homePage.checklist.${type}.title`)}
+        </Element>
+        <Heading
+          center
+          size="medium"
+          border="topCenter"
+          className="checklist__headline"
+        >
+          {t(`loggedOutArea.homePage.checklist.${type}.headline`)}
+        </Heading>
+        <Columns>
```

Comment:
Columns are now just making the code harder to read and don't bring any benefit. Please remove them.

Reply:
`Column` deleted and css for `Element` adjusted accordingly